backend/include/Options.hpp

```diff
@@ -10,6 +10,7 @@ class Options
         Options();
         std::vector<Palette> Palettes; ///< All saved palettes.
         size_t activePalette; ///< Index of currently active palette inside palettes vector.
+		size_t canvasSize; ///< size of canvas in pixel.
```

Comment:
Already stored in WGLContext?

Reply:
Thanks for the hint ^^ Totally forgot that